### PR TITLE
Chapman Opcodes

### PR DESCRIPTION
**Pull request by TseChapman**

### CODE REVIEW

File: disassembler.x68 [hunk 1]
```diff
@@ -1,653 +1,673 @@
...
+        MOVE.L  STADR,A2    ; load starting address
+LOOPMEM:
+        MOVE.W  (A2),D2    ; each instruction is at least a word in machine code
+        * Do action here *
+DECODENOP:
+        MOVE.W  D2, D3      ; make a copy in d3 to run tests on the copy
+        EORI.W  #$4E71,D3   ; NOP XOR NOP would equal 0
+        CMP.W   #0,D3
+        BEQ     PRINTNOP
+DECODERTS:
```
Comment (TseChapman) [inline disassembler.x68:161]: New function here

File: disassembler.x68 [hunk 2]
```diff
@@ -1,653 +1,673 @@
...
+        MOVE.B  #14,D0     
+        TRAP    #15
+        JSR     PRINTNEWLINE
+        JSR     CLEAR_ALL
+        MOVE.W  (A2)+,D2    ; address should be incremented at the end of each print
+        CMP.L   ENADR,A2   ; keep looping until reach the end
+        BLT     LOOPMEM
+        BRA     DONE
+
+PRINTRTS:
```
Comment (TseChapman) [inline disassembler.x68:356]: RTS print function

**Review by Leoism — CHANGES_REQUESTED** [review]
Before you merge, could you pull the .gitattributes file from master to prevent a full file change and to make merge conflicts in the future easier to handle?

**Comment by Leoism** [comment]
> Before you merge, could you pull the .gitattributes file from master to prevent a full file change and to make merge conflicts in the future easier to handle?

Nevermind on that, it looks like you already have. I'll have to look at this issue closer

**Pull request merged**